src/app/pages/AddEntry/AddEntry.tsx

```diff
@@ -0,0 +1,37 @@
+import styles from './AddEntry.module.css';
+import Button from '../../components/Button/Button';
+import Navigation from '../../components/Navigation/Navigation';
+import UploadImageIcon from '../../components/UploadImage/UploadImageIcon.svg';
+
+function AddEntry(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <Navigation headerTitle={'How was your day?'} />
+      <form className={styles.form}>
+        <input
+          className={styles.titleInput}
+          type="text"
+          placeholder="... add title ..."
+          required
+        ></input>
```

Comment:
Do you need this closing tag here?
```suggestion
        / >
```